web-components/src/components/inputs/new/LocationPicker/LocationPicker.tsx

```diff
@@ -129,8 +125,14 @@ export const LocationPicker = ({
         </div>
         {/* We simply absolutely position the location pin at the center,
           without using a react-map-gl Marker, so the map moves around the pin */}
-        <div className="absolute top-[50%] left-[50%]">
+        <div className="absolute top-[50%] left-[50%] -ml-[18.5px] -mt-50">
```

Comment:
Same here, while the impact is minimal too, there are built-in utilities for `left-[50%]` and `top-[50%]`:  `left-1/2` and `top-1/2`. Using built-in utilities helps reduce the size of the generated CSS.